agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/DiseaseAnnotationCurationIndexer.java

```diff
@@ -116,7 +116,11 @@ private List<GeneDiseaseAnnotationDocument> createGeneDiseaseAnnotationDocuments
 				}
 				gdad.setEvidenceCodes(da.getEvidenceCodes());
 				if(CollectionUtils.isNotEmpty(da.getDiseaseQualifiers())) {
-					gdad.setDiseaseQualifiers(da.getDiseaseQualifiers().stream().map(term -> diseaseQualifierMapping.get(term.getName())).collect(Collectors.toSet()));
+					log.info(gdad.getSubject().getCurie());
+					Set<String> diseaseQualifiers = da.getDiseaseQualifiers().stream().map(term -> diseaseQualifierMapping.get(term.getName())).collect(Collectors.toSet());
+					log.info("No. of qualifiers: "+diseaseQualifiers.size());
+					gdad.setDiseaseQualifiers(diseaseQualifiers);
+					log.info(gdad.getDiseaseQualifiers().iterator().next());
```

Comment:
@cmpich do these logs need to be in here?

Reply:
Just for debugging now...

Reply:
Can you put in a tick to remove these log lines.